README.md

```diff
@@ -141,6 +141,7 @@ See the complete list of options below.
 |`check_name`|`"Unit Test Results"`|An alternative name for the check result.|
 |`comment_title`|same as `check_name`|An alternative name for the pull request comment.|
 |`comment_mode`|`update last`|The action posts comments to a pull request that is associated with the commit. Set to `create new` to create a new comment on each commit, `update last` to create only one comment and update later on, `off` to not create pull request comments.|
+|`job_summary`|`false`| When set to `true` the report summary is printed as markdown in file [`$GITHUB_STEP_SUMMARY`](https://docs.github.com/en/actions/learn-github-actions/environment-variables#default-environment-variables)|
```

Comment:
I am happy to have this default to `true`!

Reply:
If default is true, should we really make it configurable?

Reply:
There is surely someone who does not want it.